phalcon/db/adapter/pdo.zep

```diff
@@ -745,7 +762,14 @@ abstract class Pdo extends Adapter
 	 */
 	public function getInternalHandler() -> <\Pdo>
 	{
-		return this->_pdo;
+		var pdo;
+		let pdo = this->_pdo;
+		if typeof pdo != "object" {
+			error_log(typeof pdo);
```

Comment:
Yes this is a working copy.  Its an easy way for me to get Travis to run tests.  There are complications and I'm not 100% sure if I have all bugs worked out of Phalcon yet and my plugin needs to be redesigned.